contracts/actions/liquity/LiquityClose.sol

```diff
@@ -44,7 +46,7 @@ contract LiquityClose is ActionBase, LiquityHelper {
     /// @param _from Address where to pull the LUSD tokens from
     /// @param _to Address that will receive the collateral
     function _liquityClose(address _from, address _to) internal returns (uint256) {
-        uint256 debt = TroveManager.getTroveDebt(address(this));
+        uint256 debt = TroveManager.getTroveDebt(address(this)).sub(200e18);
```

Comment:
Don't hardcode magic values, set on top of file as constant which will be an explanation of why we sub it.

Reply:
Also name of the variable should then be diff.